pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,56 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitmenty
+author: Core Team, Edited
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+Alongside the protesters in the US and around the world, the Mixxx team unequivocally renounces police brutality.
+The systemic racism that burdens people of color every day is the responsibility of white people around the world to address.
+Mixxx leadership has been discussing concrete actions we can take as a community and today we announce two efforts: first, to eliminate "master" and "slave" terminology from our code base and second, to recruit paid interns from underrepresented communities to join our contributor base.
+
+## Terminology Changes
+
+Major projects like [Python](https://www.vice.com/en_us/article/8x7akv/masterslave-terminology-was-removed-from-python-programming-language), [Go](https://www.reddit.com/r/golang/comments/gy9ylr/go_has_removed_all_uses_of_blacklistwhitelist_and/fte1zk0/), and now [Github](https://www.cnet.com/news/microsofts-github-is-removing-coding-terms-like-master-and-slave/) have all worked to eliminate the use of the problematic terms, "master" and "slave," and today Mixxx joins them.
+The issue of nomenclature and terminology in software is one example of how deeply embedded white privilege is around the globe and we hope changing these terms is one small step towards a more inclusive and diverse coding community.
+
+Although we eliminated the use of the word "slave" back in [2016](https://github.com/mixxxdj/mixxx/commit/e59916caf72a256bb28b1722759a629c5cc9cf81), "master" still appears in a number of places.
+We have a "master" code branch, a "Master Sync" feature, an "EngineMaster" object, and more.
+We are working to migrate our code away from these words, even in cases where the particular meaning of "master" was never explicitly associated with a problematic context.
+
+It’s important to note that there will be some cases where the word will remain.
+Many DJ controllers have buttons marked "master" on them, and we need to refer to those buttons by name.
+There are also the audio engineering concepts of a "master tape" or "audio mastering", so there may be areas of our documentation or code that would be less understandable if we tried to use synonyms.
+But in situations where master is used in code and previously was part of a master/slave terminology pairing, we commit to changing the language we use by DATEDATEDATE **TKTKTKTK commit to a timeline to change the main branch?? TKTKTKTKTK**.
+
+But changing language is not enough.
+
+## Diversifying Recruiting
+
+In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either American or European.
```

Comment:
```suggestion
In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either from northern America or Europe.
```
in case you refered to the US here

Reply:
"North American"? Is there a Canadian contributor you want to mention, or should I say "from the United States"?

Reply:
Don't forget Greenland :wink: I think we can use "United States" here.

Reply:
Agreed that "from the US or Europe" (and maybe western Europe specifically?) is clearer.

Reply:
fixed

Reply:
fixed